caim_base/views/browse.py

```diff
@@ -27,9 +28,15 @@ def parse_euth_date(args):
 
 
 def view(request):
-    if request.GET.get("animal_type") == "CAT":
+
```

Comment:
It might need testing but you should be able to get away with just using the base wrapper for everything, is there a specific reason you're keeping that `partial_render.html`?